docs/react-testing-library/setup.mdx

```diff
@@ -283,15 +283,19 @@ using Create React App without TypeScript, save this to `jsconfig.json` instead.
 }
 ```
 
-### Jest 27
+### Jest 28
```

Comment:
What do you think of keeping the Jest 27 docs and adding a new section for Jest 28?
Because Jest 27 is still popular (and other Jest versions are also included).

Reply:
Sure.
I initially thought to do that but the terminology was getting confusing and duplicated. E.g. 27 or later, the docblocks are duplicated.
I have updated it now. Let me know what you think